configs/LLNL-Magma-Penguin-icelake-OmniPath/caliper_topdown.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+variables:
+  modifier_package_name: 'caliper-topdown'
```

Comment:
Please remove from this PR, lets only have the platform independent modifier for now.